scripts/exrrfs_run_prdgen.sh

```diff
@@ -196,6 +196,7 @@ natlev=${net4}.t${cyc}z.natlev.f${fhr}.${gridname}grib2
 ififip=${net4}.t${cyc}z.ififip.f${fhr}.${gridname}grib2
 testbed=${net4}.t${cyc}z.testbed.f${fhr}.${gridname}grib2
 
+
```

Comment:
You can just check out `scripts/exrrfs_run_prdgen.sh` from the latest dev-sci branch. No changes are needed in this file.

Reply:
@EdwardColon-NOAA There are no real changes in this file. Please check out this file from the lastest dev-sci branch and replace what in your branch.